apps/website/src/content/docs/tabs.mdx

```diff
@@ -59,7 +59,7 @@ You can build Tabs with subcomponents that are fully customizable.
 
 Alternatively, you can use Tabs directly with labels, actions, and children as panels. Although this pattern can suffice for simple cases, we strongly recommend using composition API for the best experience.
 
-The following code is equivalent to the [composition example](#composition-api) above:
+The following code is (mostly) equivalent to the [composition example](#composition-api) above:
```

Comment:
nit: Would making "(mostly)" italic be better?

```suggestion
The following code is _(mostly)_ equivalent to the [composition example](#composition-api) above:
```

Reply:
italics for emphasis, i don't think we want to emphasize it here